handbook/reference/problems/actor-validation-failed.md

```diff
@@ -0,0 +1,29 @@
+# 400 Actor Validation Failed
+
+The server will not process the request due to validation issues around actor delegations.
+
+The client can try again after delegating to correct delegation.
+
+## Extra metadata
+
+### `fields`
+
+A JSON object listing validation issues where delegatedUser is the nationalId of the user assigned to the resource that the actor is a and actor is the nationalId of the user trying to access the resource.
```

Comment:
> resource that the **actor is a and** actor is the nationalId of the user trying to access the resource.
Don't understand this. Is something missing?

Reply:
@alexandradilja perhaps? :D